xds/internal/balancer/cdsbalancer/cluster_handler_test.go

```diff
@@ -719,6 +734,110 @@ func (s) TestAggregatedCluster_NodeChildOfItself(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// Verify that a successful RPC can be made.
+	if _, err := client.EmptyCall(ctx, &testpb.Empty{}, grpc.WaitForReady(true)); err != nil {
+		t.Fatalf("EmptyCall() failed: %v", err)
+	}
+}
```

Comment:
Here and elsewhere in the test: I don't know if there is, but can you assert on the address somehow?

Reply:
Why is that assertion useful here? Cluster A does not have any endpoints, and cluster B has a single endpoint. If the RPC succeeds, it *must* have reached that single endpoint.